src/GobanCore.ts

```diff
@@ -87,8 +87,7 @@ export type GobanModes =
     | "analyze"
     | "conditional"
     | "setup"
```

Comment:
It seems like "setup" and "edit" are similarly unused.  There is no logic attached to "setup", so I think this may have been due to confusion with the puzzle_mode setup. "edit" does have logic, but is unused in the other repos.